action.yml

```diff
@@ -11,6 +11,9 @@ inputs:
   restore-keys:
     description: 'An ordered list of keys to use for restoring the cache if no cache hit occurred for key'
     required: false
+  upload-chunk-size:
+    description: 'The chunk size used when uploading large cache files'
```

Comment:
We should include the unit in either the description or input name (`upload-chunk-size-bytes`)